src/gausskernel/storage/access/k2/storage.cpp

```diff
@@ -126,6 +126,9 @@ static void populateSysColumnFromSKVRecord(skv::http::dto::SKVRecord& record, in
             syscols->k2pgbasectid = (uint8_t*)PointerGetDatum(datum);
         }
             break;
+        case PgSystemAttrNum::kPgRowId:
+            // PG does not want to read rowid back
```

Comment:
seems need to skip this record in skv record, otherwise, the cursor isn't correct, right?